pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -855,25 +856,27 @@ func (m *kubeGenericRuntimeManager) purgeInitContainers(ctx context.Context, pod
 // index of next init container to start, or done if there are no further init containers.
```

Comment:
Is the meaning of "done" (as in "initialization is complete, you can stop calling this function") still valid?  Or does this mean something more like "nothing to do right now" ("but if you call this function again later it might find some sidecars to restart")?

Apologies if I'm misreading the code.

Reply:
"done" means "nothing to do right now for this sync" now.

Reply:
Revise this comment to match?  This code is high density, probably worth it.

Reply:
done.